slim.Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Build wheels
+FROM python:3.9-slim as build
+
+# Need git for setuptools_scm
+RUN apt-get update \
```

Comment:
If we are using setuptools_scm, we probably don't want `.git` in the `.dockerignore` file, as we won't pick up the version correctly.

Reply:
I took `.git` out of the `.dockignore` for this